corehq/apps/reports/util.py

```diff
@@ -248,6 +248,11 @@ def parts():
         return info
 
 
+def get_paginated_user_query(user_es_query):
+    users_query = user_es_query.fields(SimplifiedUserInfo.ES_FIELDS).scroll()
+    return users_query
```

Comment:
I think I'd do this in-line - it's part of the API already, not sure wrapping it adds much.